static/app/views/organizationContext.tsx

```diff
@@ -16,6 +16,15 @@ import useApi from 'sentry/utils/useApi';
 import {useParams} from 'sentry/utils/useParams';
 import {useRoutes} from 'sentry/utils/useRoutes';
 
+interface OrganizationLoaderContextProps {
+  isLoading: boolean;
+  loadOrganization: () => void;
```

Comment:
it might be cool to have this return a `Promise` rather than void, then you wouldn't need to maintain a separate `isLoading` variable, instead you could just continue the promise chain and use the promises state to for the load state.

Reply:
🔥 i'd say this is like 99% there, i left a comment on the `useEffect` but i think you can basically remove it and be good to go.